core/schema/src/main/resources/org/hawkular/schema/updates/schema-0.19.0.groovy

```diff
@@ -24,4 +24,70 @@ schemaChange {
   cql """
 ALTER TABLE data WITH COMPRESSION = {'sstable_compression': 'DeflateCompressor'};
 """
-}
\ No newline at end of file
+}
+
+schemaChange {
+  version '3.1'
+  author 'jsanda'
+  tags '0.19.x'
+  cql """
+CREATE TABLE rollup60 (
+    tenant_id text,
+    metric text,
+    shard  bigint,
+    time timestamp,
+    min double,
```

Comment:
>What about map of aggregates? 
What exactly do you have in mind?

>Also, just one table for rollup60, rollup300, rollup3600 is necessary in my opinion.

This is something I've gone back and forth with for data modeling in general with Cassandra. Of the other Cassandra based systems I have looked at, the majority break things out into multiple tables. What is the basis for your opinion?